modules/backend/clinic/Clinic.cpp

```diff
@@ -0,0 +1,14 @@
+#include "Clinic.hpp"
+
+#include "room/iRoom.hpp"
+#include "staff/Staff.hpp"
```

Comment:
Do I understand it correctly, that because of CMake with CMAKE_CURRENT_SOURCE_DIR under `modules/backend/clinic` it is not necessary to use  `modules/backend/clinic` in the includes? 
Or it could be used? Or it will look weird like  `../../../modules/backend/clinic/'?

Reply:
Yes. it is like that.
In general, In current solution I made every module top level dir accesible. And including anything inside module directly resembles it's own structure.
For ex. if we are in Clinic, we simply include room/iRoom.hpp because this is folder structure.

Conversely, if we are in different module like werehouse and in Warehouse.hpp (or even deeper), if we want something from clinic, like room, we are only interested in top level dir of clinic (which is implicitly provided).
So there we only write #include "room/iRoom.hpp" as well.

module/backend/clinic in general no longer works.
Absolute paths should work (but no sense). Relative paths will most likely work as well

Reply:
So we only need to remember where top module dirs are.
This could be easily seen in installed headers in own_libraries
![image](https://github.com/coders-school/desktop-business-app/assets/82645568/afcd460a-c688-4b77-a5d6-2aafeb5ac6a9)